### ISSUE TITLE
"conduct setup-dcos" does not generate package.json

### ISSUE THREAD

**Issue by yuchaoran2011** [typesafehub/conductr-cli#512]
After running `conduct setup-dcos`, the following command:
`dcos package list`
gives an error: 
```
Error opening file [/home/ubuntu/.dcos/subcommands/conductr/package.json]: No such file or directory
```
Only the `env` folder is under ~/.dcos/subcommands/conductr/ but nothing else.

**Comment by huntc** [comment]
@markusjura this may go away magically with universe right?

**Comment by yuchaoran2011** [comment]
Any idea when this can be fixed? @huntc @markusjura

**Comment by huntc** [comment]
@yuchaoran2011 Is this still a problem - I've not heard of others having it...

**Comment by yuchaoran2011** [comment]
I just verified that the error still exists. @deanwampler initially discovered it. I'll let him comment on when we need this for a future FDP release.

**Comment by deanwampler** [comment]
Too late for Friday's release, but can we get it fixed in the next few weeks?

**Comment by huntc** [comment]
@longshorej Can you please also confirm on your DC/OS test cluster?

**Comment by yuchaoran2011** [comment]
Now that we are using DC/OS 1.10, the DC/OS UI no longer offers the option to directly destroy a service. When one wants to delete conductr, the UI simply shows a dialog box prompting the user to run ` dcos package uninstall conductr --app-id=/conductr`.

This is what I got when running the command:

ubuntu@cyubootstrap:~$ dcos package uninstall conductr --app-id=/conductr
WARNING: This action cannot be undone. This will uninstall [/conductr] and delete all of its persistent (logs, configurations, database artifacts, everything).
Please type the name of the service to confirm: /conductr
Error opening file [/home/ubuntu/.dcos/subcommands/conductr/package.json]: No such file or directory

The user can still uninstall conductr by running `dcos marathon app remove conductr`. But we need to fix the error above to improve user experience.

**Comment by longshorej** [comment]
@yuchaoran2011 Not sure if this is relevant for you guys anymore but this is fixed with the CLI release `1.2.23`, after you run `conduct setup-dcos`.
